<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GCharacterObserver</class>
 <widget class="QMainWindow" name="GCharacterObserver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GCharacterObserver</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="infoBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Info</string>
    </property>
    <widget class="QLabel" name="nameLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameField">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="levelLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Level:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="levelField">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="attributesGroup">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>191</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Attributes</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>71</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="attLabelVBox">
      <item>
       <widget class="QLabel" name="strLabel">
        <property name="text">
         <string>Strength:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="dexLabel">
        <property name="text">
         <string>Dexterity:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="conLabel">
        <property name="text">
         <string>Constitution:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="intLabel">
        <property name="text">
         <string>Intelligence:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="wisLabel">
        <property name="text">
         <string>Wisdom:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="chaLabel">
        <property name="text">
         <string>Charisma:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>52</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="attScoreVBox">
      <item>
       <widget class="QLineEdit" name="strField">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dexField">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="conField">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="intField">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="wisField">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="chaField">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>31</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="attModVBox">
      <item>
       <widget class="QLineEdit" name="modStrField">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="modDexField">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="modConField">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="modIntField">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="modWisField">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="modChaField">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="scoreLabel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Score</string>
     </property>
    </widget>
    <widget class="QLabel" name="modLabel">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mod</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="invBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>351</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Inventory</string>
    </property>
    <widget class="QTextBrowser" name="invField">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>331</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
